### PR TITLE
fix lint

### PR DESCRIPTION
**Pull request by neo773**
Screenshot 2023-11-04 at 2 43 20 PM

/claim #572

### CODE REVIEW

File: lint.sh [hunk 1]
```diff
@@ -41,11 +44,19 @@ fi
...
     *)
         echo "Invalid mode. Please use --mode=check or --mode=fix"
         exit 1
         ;;
 esac
+
+# If any command failed, exit with a non-zero status code
+if [ $FMT_EXIT_CODE -ne 0 ] || [ $CLIPPY_EXIT_CODE -ne 0 ] || [ $PRETTIER_EXIT_CODE -ne 0 ]; then
+    exit 1
+fi
```
Comment (tusharmath) [inline lint.sh:62]:
```suggestion
fi

```

Comment (tusharmath) [inline lint.sh:62]:
```suggestion
if (( (FMT_EXIT_CODE | CLIPPY_EXIT_CODE | PRETTIER_EXIT_CODE) != 0 )); then
    exit 1
fi
```

Will this work?

**Comment by tusharmath** [comment]
@neo773 Can you comment on the issue stating that you are going to attempt, so that we don't get a lot of attempts.

Secondly, can you fix the errors also, for a tip?

**Comment by neo773** [comment]
@tusharmath 
Fixed, CI is passing now

**Review by tusharmath — APPROVED** [review]

**Pull request merged**